entity-web/src/main/java/eu/europeana/entity/web/jsonld/JsonLdSerializer.java

```diff
@@ -0,0 +1,32 @@
+package eu.europeana.entity.web.jsonld;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.europeana.api.commons.definitions.statistics.entity.EntityMetric;
+import eu.europeana.entity.app.AppConfigConstants;
+import eu.europeana.entity.web.controller.exception.EntityApiRuntimeException;
+import org.springframework.stereotype.Component;
+
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+@Component(AppConfigConstants.BEAN_EM_JSONLD_SERIALIZER)
+public class JsonLdSerializer {
+
+    ObjectMapper mapper;
+    public static final String DATE_FORMAT  = "yyyy-MM-dd'T'HH:mm:ss'Z'";
+
+    public JsonLdSerializer() {
+        mapper = new ObjectMapper();
+        SimpleDateFormat df = new SimpleDateFormat(DATE_FORMAT, Locale.ENGLISH);
+        mapper.setDateFormat(df);
+    }
+
+    public String serializeMetric(EntityMetric metric) throws EntityApiRuntimeException {
```

Comment:
better use a generic implementation public String serialize(Object) ... as the implementation doesn't depend on the type of the object

Reply:
done